adafruit_emc2101/__init__.py

```diff
@@ -255,7 +347,7 @@ def fan_speed(self):
 
         val = self._tach_read_lsb
         val |= self._tach_read_msb << 8
-        return _FAN_RPM_DIVISOR / val
+        return round(_FAN_RPM_DIVISOR / val, 2)
```

Comment:
I don't know the use case for having the library use `round()` as opposed to usercode, is there a reason this should be done here?  Is there any reason a user might want more precision?

If it should be rounded, the product tutorial has an example with 3 decimal places, is that better?

What are your thoughts @rivimey?

Reply:
The sensors have an accuracy, which is probably less than 1dp. The "additional" dp's present in the number are entirely mythical, arising from the division. I feel it is helpful to users not to present an apparent resolution far higher than the actual precision.

The reason for the low precision of the reading is partly that motors habitually fluctuate in speed around their nominal value, so any frequency measurement over time is an average; the pwm also has a resolution, and of course all these readings are in fact historical.